packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -70,12 +70,28 @@ export class Accordion extends FoundationElement {
         this.$emit("change");
     };
 
+    private findExpandedItem(): AccordionItem | null {
+        for (let item: number = 0; item < this.accordionItems.length; item++) {
+            if (this.accordionItems[item].hasAttribute("expanded")) {
+                return this.accordionItems[item] as AccordionItem;
+            }
+        }
+        return null;
+    }
+
     private setItems = (): void => {
+        if (this.accordionItems.length === 0) {
+            return;
+        }
         this.accordionIds = this.getItemIds();
```

Comment:
You can get rid of the `this.accordionIds = this.getItemIds();` line in the `accordionItemsChanged` function as that function immediately calls this one, so we're getting item id's more than we need to.  Not specifically part of your change but since you're in the neighborhood...

Reply:
removed